routes/index.js

```diff
@@ -5,6 +5,7 @@ app.use('/healthcheck', require('./healthCheck.js'))
 app.use('/auth', require('./auth.js'))
 app.use('/users', require('./users.js'))
 app.use('/members', require('./members.js'))
+app.use('/roadmap-site', require('./roadmap-site/index'))
```

Comment:
Are we still having `roadmap` site?

Reply:
We will take care of this going forward @swarajpure